### PR TITLE
Feature Config Deploy To Fabric & Single Log on

### PR DESCRIPTION
**Pull request by daolq3012**
- Config deploy to fabric
- Single log on

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveActivity.java [hunk 1]
```diff
@@ -1,14 +1,19 @@
 package com.framgia.wsm.screen.requestleave;
 
+import android.app.Dialog;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveActivity.java]: ERROR: Unused import: android.app.Dialog. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveActivity.java]: WARNING: [UnusedImports] Avoid unused imports such as 'android.app.Dialog'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnusedImports

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -278,4 +278,5 @@
     <string name="create_form_success">Create form success!</string>
     <string name="edit_form_success">Edit form success!</string>
     <string name="delete_form_success">Delete form success!</string>
+    <string name="this_account_has_been_login_in_another_device">This account has been login in another device.</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`this_account_has_been_login_in_another_device`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/java/com/framgia/wsm/screen/login/LoginActivity.java [hunk 3]
```diff
@@ -1,17 +1,18 @@
 package com.framgia.wsm.screen.login;
 
+import android.content.Intent;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/login/LoginActivity.java]: ERROR: Unused import: android.content.Intent. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/login/LoginActivity.java]: WARNING: [UnusedImports] Avoid unused imports such as 'android.content.Intent'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnusedImports

**Comment by daolq3012** [comment]
OK!

**Pull request merged**